ucm2/amd-soundwire/rt722-sdca.conf

```diff
@@ -0,0 +1,81 @@
+# Use case Configuration for amd-soundwire card
+
+If.RT722 {
```

Comment:
The config is almost identical to ucm2/sof-soundwire . I would add/merge code. If AMD driver follow the similar components scheme (`${CardComponents}` string), it can be a good benefit to share configs.

Reply:
@perexg : Agreed. But due to time constraint, We need to release the UCM files on urgent basis to unblock the customer release.  Let this PR  go as initial version. We will look in to your suggested changes and implement the changes as next version.

Reply:
Could you show the components string ? `amixer -c 0 info` will print it (replace zero with the correct ALSA card number).